socorro/external/es/super_search_fields.py

```diff
@@ -358,26 +358,26 @@ def keyword_field(
     }
 
 
-def number_field(
+def integer_field(
     name,
     namespace="processed_crash",
     in_database_name="",
     number_type="integer",
 ):
-    """Generates a numeric field.
+    """Generates a whole number field.
 
     :param name: the name used to query the field in super search
     :param namespace: either "raw_crash" or "processed_crash"; note that we're moving
         to a model where we pull everything from the processed_crash, so prefer that
     :param in_database_name: the field in the processed crash to pull this data from
-    :param number_type: "short", "integer", "long", "double"
+    :param number_type: "short", "integer", "long"
```

Comment:
I should fix this--it should talk about what `number_type` affects.

Reply:
We might even want to rename it to `storage_mapping_type`. That's infinitely clearer.